README.md

```diff
@@ -237,28 +237,25 @@ We have automatic builds set up, so commits to master will trigger a build of ht
 
 ### Development usage
 
-If you want to use Docker in development, then you can build it locally with:
+If you want to use Docker in development, you can build and run the image locally with either docker-compose:
```

Comment:
Colon is not correct here. M-dash is.
```suggestion
If you want to use Docker in development, you can build and run the image locally with either docker-compose —
```

Reply:
Ok. I replaced colon by M-dash. Please, check it's ok now. Tks.